### PR TITLE
password review feature

### PR DESCRIPTION
**Pull request by Aimanosi**
**What kind of change does this PR introduce?**
password review feature on the log-in modal page.

**Issue Number:**
Fixes issue
- #612 

**Did you add tests for your changes?**
No

**Snapshots/Videos:**

https://user-images.githubusercontent.com/104039356/224178475-63b82528-b34b-4e28-88ed-417e824e487e.mp4

**If relevant, did you update the documentation?**

**Summary**
I created a label tag with text "show" on the right side of the password input field and attached a toggle onclick function to display the password when it is clicked so that users can confirm their password before they log-in.

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by Aimanosi** [comment]
HI @palisadoes This PR is ready for review, please let me know if you want me to do anything else, thank you.

**Comment by palisadoes** [comment]
Please fix the first comment so that the issue automatically closes. The PR_GUIDELINES.md file has details.

**Comment by palisadoes** [comment]
Please add tests for the lines mentioned here:

- https://github.com/PalisadoesFoundation/talawa-admin/pull/629/files

We need to maintain our reliability

**Review by palisadoes — CHANGES_REQUESTED** [review]
Please add tests for the lines mentioned here:

- https://github.com/PalisadoesFoundation/talawa-admin/pull/629/files

We need to maintain our reliability

**Comment by Aimanosi** [comment]
Hi Peter,

Thank you so much for the help, I made a PR and was told to do a review,
which is adding tests to a line in my code, i have never done this before
so I am a little confused on how to start the process, a little help from
you will mean the world to me, thank you.

Find attached the images from the changes requested.

**Comment by Aimanosi** [comment]
HI @palisadoes This PR is ready for review, please let me know if you want me to do anything else, thank you.

**Review by palisadoes — APPROVED** [review]

**Review by noman2002 — APPROVED** [review]

**Review by Tajcore — DISMISSED** [review]
Lgtm

**Review by palisadoes — DISMISSED** [review]
Please fix the failing tests

**Comment by Aimanosi** [comment]
Hi @palisadoes @Tajcore @noman2002 @heyjordn 

Thanks for your help thus far, however I'm finding it quite challenging to solve this new test fail, Initially I wrote a test to include the changes I made in the codebase and that gave me a 100% coverage on the "loginPage.tsx" and it passed the first test fail I had, could the test fails in the other files I didn't work on be responsible for this new fails(
PR Workflow / Shows code coverage (pull_request))? if so, how do I resolve this?
terminal result 001
terminal result 02


Thank you.

File: src/screens/LoginPage/LoginPage.test.tsx [hunk 1]
```diff
@@ -323,4 +323,34 @@ describe('Testing Login Page Screen', () => {
 
     await wait();
   });
+
+  test('Testing password preview feature', async () => {
+    const { getByTestId } = render(
```
Comment (kayceeDev) [inline src/screens/LoginPage/LoginPage.test.tsx:328]: Why are yoy destructuring `getByTestId`? Won't `screen.getByTestId` work?

**Comment by Aimanosi** [comment]
Hi @palisadoes please review this, thank you.

File: src/screens/LoginPage/LoginPage.module.css [hunk 2]
```diff
@@ -2,7 +2,18 @@ html,
 body {
   overflow-x: hidden !important;
 }
+
 /* ===== Login Modal CSS Start ===== */
+
+/* = Nasiru's hide and show password style = */
```
Comment (kayceeDev) [inline src/screens/LoginPage/LoginPage.module.css:8]: Could you remove this comment? It is unnecessary and doesn't do anything. Also, this is a public repository and it is not advised to add your name to the comment. The git commit with your name is enough. Make sure there is a consistent line break.

File: src/screens/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -40,6 +40,7 @@ function LoginPage(): JSX.Element {
     email: '',
     password: '',
   });
+  const [show, setShow] = useState(false);
```
Comment (kayceeDev) [inline src/screens/LoginPage/LoginPage.tsx:43]: Since we are using typescript, you should type this e.g `useState<boolean>(false)`

**Review by kayceeDev — CHANGES_REQUESTED** [review]

**Review by kayceeDev — APPROVED** [review]
Well done!! LGTM✅

**Comment by Aimanosi** [comment]
Hi @palisadoes please review this, thank you.

**Comment by Aimanosi** [comment]
Hi @palisadoes @Tajcore @noman2002 
Please review this, thank you.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**